qkeras/quantizers.py

```diff
@@ -1706,6 +1706,80 @@ def get_config(self):
     }
     return config
 
+class quantized_sigmoid(BaseQuantizer):  # pylint: disable=invalid-name
+  """Computes a quantized sigmoid to a number of bits.
+
+  Modified from:
+
+  [https://github.com/BertMoons/QuantizedNeuralNetworks-Keras-Tensorflow]
+
+  Attributes:
+    bits: number of bits to perform quantization.
+    integer: number of bits to the left of the decimal point.
```

Comment:
Why do we need integer bits? The largest quantized value that we have with sigmoid is 1.0 by definition. So we would at most only need one integer bit, but 0 integer bits may also be enough if we have enough fractional bits.

Reply:
Integer part removed